ReSharper.FSharp/src/FSharp.Psi/src/Impl/Tree/LocalBinding.cs

```diff
@@ -18,5 +24,33 @@ public void SetIsMutable(bool value)
       if (headPat != null)
         FSharpImplUtil.AddTokenBefore(headPat, FSharpTokenType.MUTABLE);
     }
+
+    /// A workaround for getting a declared element for binding in features like Find Usages results and
+    /// file member navigation where we're looking for containing type member.
```

Comment:
The binding types currently don't implement it due to it'd be more difficult to work with in cases with multile declared elements like:
```fsharp
let x, y = 1, 2
let (UnionCase(fieldA, fieldB) = Foo
```
Please don't add these members to binding types, at least of a part of this PR where it seems unrelated.